### DISCUSSION TITLE
Add ani option to hide shadow on current possition mark

### DISCUSSION THREAD

**Discussion by thuryn** [osmandapp/OsmAnd#17822] [Ideas]
I would like to have an option to hide shadow under the current possition mark - it does not make sense for me a hides details around the current possition point.

**Comment by DmitryAlexei** [comment]
Could you, please, add screenshot of your issue and info about your version of the app? (Menu > Help > about > long tap copies the data)

**Comment by thuryn** [comment]
![photo_2023-08-09_12-39-13](https://github.com/osmandapp/OsmAnd/assets/11028346/0fd8c821-3229-4655-8a65-cbaa94ddcdea)

**Reply by DmitryAlexei** [reply]
I can't see the screenshot, please upload it again

**Reply by thuryn** [reply]
but it is not important - the important thing is that the shade hides map details in some cases and I don't want it anyway - it's useless from my point of view

**Comment by thuryn** [comment]
It is OK here - it's just for explanation. Otherwise the shade covers interresting information and it useless from my point of view

**Reply by DmitryAlexei** [reply]
Should have been fixed already, at least in the latest nightly builds. Soon (in 4.6) will be available in Google play. https://github.com/osmandapp/OsmAnd/issues/16787

**Comment by pebogufi** [comment]
Not fixed, shadow remains in OsmAnd~ 4.6.0#628m, veröffentlicht: 2023-08-17
Samsung tab S5e, Android 10
![Screenshot_20230817-165537_OsmAnd~~01](https://github.com/osmandapp/OsmAnd/assets/18628183/6b1d14d3-1c12-48a6-81c2-ea26e5fa0b15)



NO shadow with Samsung S10, Android 12.OsmAnd~ 4.6.0#628m

As you can see here
https://github.com/osmandapp/OsmAnd/issues/16787#issuecomment-1500311583

Icons 5 and 6 did not have a shadow,
NOW, since some weeks icon 5 also has a shadow.

**Reply by dmpr0** [reply]
On the map they look the same?

**Reply by pebogufi** [reply]
Yes

![Screenshot_20230817-172205_OsmAnd~](https://github.com/osmandapp/OsmAnd/assets/18628183/ddd45380-6bd3-4a4d-962b-0a6331c21acf)
